go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/companieshouse/chs-delta-api
 
-go 1.12
+go 1.15
 
 require (
-	github.com/companieshouse/chs.go v1.2.1
+	github.com/companieshouse/chs.go v1.2.4
 	github.com/companieshouse/gofigure v0.1.4
 	github.com/gorilla/mux v1.7.3
```

Comment:
Can we use 1.8.0 for gorilla/mux ?

Reply:
I will update this. Also to address the comment below this one, I will add to the description of this PR a link to the docker-chs-development PR when it is created.